assets/src/dashboard/components/cardGallery/index.js

```diff
@@ -153,7 +153,7 @@ function CardGallery({ story, isRTL, galleryLabel }) {
             const isActive = isCurrentPage && isInteractive;
             return (
               <ItemContainer
-                key={`page-${index}`}
+                key={page.id}
```

Comment:
I think we need to prefix this. There will a couple of places we use page id as the key.

Reply:
Nope. The key only needs to be unique for this particular sub-tree, not for the whole app.